app.js

```diff
@@ -13,324 +13,287 @@ if (!supportsLocalStorage()) {
-      .className = 'name';
-    appendToLI('label', 'textContent', 'Attending')
-      .appendChild(createElement('input', 'type', 'checkbox'));
-    appendToLI('label', 'textContent', 'Not Attending')
-      .appendChild(createElement('input', 'type', 'checkbox'));
-    appendToLI('span', 'textContent', 'Notes...')
-      .className = 'notes';
-    appendToLI('button', 'textContent', 'Edit');
-    appendToLI('button', 'textContent', 'Remove');
-    return li;
+  //Switches state between editing and saved modes, coordinates info that needs to pass through the state change. Syncs edits with all lists.
+  const editNotes = (invitee, eventTarget) => {
+    fullInviteList.map((current, index) => {
+      let input = eventTarget.previousElementSibling.firstElementChild.value;
+      if (invitee === current.name.replace(/\s/g, '') && current.editState === 'Saved') {
+        current.editState = 'Editing';
+      } else if (invitee === current.name.replace(/\s/g, '') && current.editState === 'Editing') {
+        current.notes = input;
+        current.editState = 'Saved';
+      }
+    });
+    renderedInviteList = []
+    fullInviteList.map((current, index) => {
+      renderedInviteList.push(current);
+    });
+    refreshLocalStorageList(fullInviteList);
+    reloadDOM();
   }
 
-  //Create and append filters to be applied to the list
-  createFilterDiv(attendingLabel, attendingCheckbox, attendingDiv, 'Attending');
-  createFilterDiv(noShowLabel, noShowCheckbox, noShowDiv, 'Not Attending');
-  createFilterDiv(respondedLabel, respondedCheckbox, respondedDiv, 'Has Responded');
-  mainDiv.insertBefore(filterConDiv, ul);
-
-  //Make the respondedLabel and respondedCheckbox do what they say they'll do
-  respondedCheckbox.addEventListener('change', (event) => {
-    uncheckFilters(attendingCheckbox, noShowCheckbox);
-    filterAttendees('responded');
-    showAllLabels();
-  });
-
-  //Make the attendingLabel and attendingCheckbox do what they say they'll do
-  attendingCheckbox.addEventListener('change', (event) => {
-    uncheckFilters(respondedCheckbox, noShowCheckbox);
-    filterAttendees('attending');
-    if (attendingCheckbox.checked === true) {
-      hideSelectedLabel();
-    } else if (attendingCheckbox.checked === false) {
-      showAllLabels();
-    }
-  });
-
-  //Make the notAttendingLabel and notAttendingCheckbox do what they say they'll do
-  noShowCheckbox.addEventListener('change', (event) => {
-    uncheckFilters(attendingCheckbox, respondedCheckbox);
-    filterAttendees('notComing');
-    if (noShowCheckbox.checked === true) {
-      hideSelectedLabel();
-    } else if (noShowCheckbox.checked === false) {
-      showAllLabels();
-    }
-  });
+  //Removes invitee from object array. Syncs all lists.
+  const removeName = (invitee) => {
+    //Map and splice both full and rendered list so you don't inadvertantly undo a filter
+    renderedInviteList.map((current, index) => {
+    	if (current.name.replace(/\s/g, '') == invitee) {
```

Comment:
You use this function a few times, best to make it its own function and call and return the value you're looking for. Ex:
```
const trim = (string) => string.replace(/\s/g, '');
```